src/db/models/organizationMeta.ts

```diff
@@ -26,6 +30,22 @@ export default (sequelize: Sequelize.Sequelize) => {
       uuid: {
         type: Sequelize.UUID,
       },
+      countGrantsFrom: {
```

Comment:
Another minor question: for the purpose of grouping related fields, would it be clearer to do this as `grantsFromCount`, `grantsToCount` etc (general category first, specific variants later)?

Reply:
I guess this comment belonged in #24

Reply:
we can handle this in another pr!